### PR TITLE
added 6 new elsys types, tests will follow

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/elsys/elt-2/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    },
+    "externalTemperature1": {
+      "title": "External temperature 1",
+      "unit": "°C",
+      "type": "number",
+      "description": "The external temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "digital": {
```
Comment (mariusgiger) [inline types/elsys/elt-2/default.schema.json:53]: digital -> reed?

File: types/elsys/elt-lite/default.schema.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    },
+    "externalTemperature": {
+      "title": "External temperature 1",
+      "unit": "°C",
+      "type": "number",
+      "description": "The external temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "digital": {
```
Comment (mariusgiger) [inline types/elsys/elt-lite/default.schema.json:53]: reed?

File: types/elsys/elt-lite/default.schema.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      "title": "Reed",
+      "description": "If either reed or waterleak is detected this equals true",
```
Comment (mariusgiger) [inline types/elsys/elt-lite/default.schema.json:55]: why either?

File: types/elsys/ems-desk/default.schema.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    "humidity": {
+      "title": "Humidity",
+      "unit": "%",
+      "type": "number",
+      "description": "The relative humidity in %",
+      "minimum": 0,
+      "maximum": 100
+    },
+    "accX": {
```
Comment (mariusgiger) [inline types/elsys/ems-desk/default.schema.json:23]: does desk really have an accelerometer?

File: types/elsys/ems-desk/meta.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "firmwareVersion": "V1.0.0",
+  "loraDeviceClass": "A",
+  "availableSensors": [
+    "Temperature",
+    "Humidity",
+    "Accelerometer",
+    "Occupancy"
+  ],
+  "outputTopics": [
+    "default",
```
Comment (mariusgiger) [inline types/elsys/ems-desk/meta.json:17]: what about occupancy?

File: types/elsys/ems-door/default.schema.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://akenza.io/elsys/ems-door/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "accX": {
```
Comment (mariusgiger) [inline types/elsys/ems-door/default.schema.json:7]: does this sensor really have an accelerometer?

**Review by mariusgiger — CHANGES_REQUESTED** [review]

File: types/elsys/ems-desk/default.schema.json (hunk 4 above)
Comment (J2thatsme) [reply types/elsys/ems-desk/default.schema.json:23]: yes

File: types/elsys/ems-door/default.schema.json (hunk 6 above)
Comment (J2thatsme) [reply types/elsys/ems-door/default.schema.json:7]: Yes

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**